### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by LBakkerTDS**
Just a test to create a pull request from my local 'feature/docs' branch to the origin's 'dev' branch.

### CODE REVIEW

File: docs/plugins/language.md [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
 
-The `language`-method should return which language to use on initialisation. This can either be a string or a Promise that resolves a
+The `language` method returns the language to be used on initialization. This can either be a string or a promise that resolves a
 string with the language code.
 
-
-```js
-export default class App extends Lightning.Component {
+```
+export default class App extends Lightning.component {
```
Comment (michielvandergeest) [inline docs/plugins/language.md:22]: Component should be written with an uppercase (there are more occurrences, commenting only once )

File: docs/plugins/language.md [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
   }
 
 }
 ```
 
-### Translations file
+### Translations File
 
-By default the Language plugin expects a `json` file, called `translations.json` in the `static` folder of your App.
+By default, the Language plugin expects a JSON translations file, called **translations.json**, which is located in the **static** folder of your App.
```
Comment (michielvandergeest) [inline docs/plugins/language.md:38]: suggestion: expects a file with translations in JSON format, called ...

**Review by michielvandergeest — COMMENTED** [review]
WIP review until `docs/plugins/language` :)

File: docs/plugins/fpscounter.md [hunk 3]
```diff
@@ -1,21 +1,20 @@
-# FPS counter
+# FPS Counter
 
-Frames per second (FPS) is an important measure for the performance of your App. A higher FPS indicates a smoother experience.
-When validating an App on a certain platform it's useful to get an indication of the FPS score.
+The number of *Frames per second (FPS)* is an important measure for the performance of your App. The higher the FPS, the smoother the experience. When you are validating your App on a certain platform, it's useful to get an indication of the *FPS score*.
 
-The SDK contains a built-in **FPS counter** that can be turned on or off via the [Platform Setting](/plugins/settings?id=platform) `showFps` in `settings.json`.
+The SDK contains a built-in *FPS counter* that can be enabled or disabled via the Platform Setting `showFps` in `settings.json`. When it is enabled, a *real-time* counter with the current FPS will be displayed as an overlay in the top left corner of your App.
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/fpscounter.md:5]: Agreed, link added.

File: docs/plugins/fpscounter.md [hunk 4]
```diff
@@ -1,21 +1,20 @@
...
+| `interval` | 500 | Refresh rate in milliseconds |
+| `log` | false | Indicates whether or not to *also* log the calculated FPS value to the console |
+| `threshold` | 1 | Minimum difference between FPS values to display / log |
 
-- **interval** - specifies the refresh rate in miliseconds (defaults to `500`)
-- **log** - whether to _also_ log the calculated FPS value to the console (defaults to `false`)
-- **threshold** - minimal difference between FPS values to display / log (defaults to `1`)
-
-```json
+```
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/fpscounter.md:17]: Added 'json' suffix.

File: docs/plugins/image.md [hunk 5]
```diff
@@ -1,79 +1,83 @@
 # Image
 
-The standard way of displaying images in Lightning is to just specify the `src`. This is the prefered way for your App's local assets (such as background, splash screen, logo and icons).
+The standard way of displaying images in Lightning is to just specify the `src`. This is the preferred way for specifying local assets (such as background, splash screen, logo and icons) of Lightning Apps.
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/image.md:3]: Replaced 'specifying' with 'your'.

File: docs/plugins/image.md [hunk 6]
```diff
@@ -1,79 +1,83 @@
...
 
-It's adviced that you optimize an App's local assets, by resizing them to the exact size and quality you will be using them. This will be beneficial for the memory usage of your App.
+It is recommended that you *optimize* the local assets of your App by resizing them to the *exact* size and quality in which you will use them. This positively affects the memory usage of your App.
 
-However when you don't have control over the images you're displaying in your App (because they come from a remote API for example), you can use the Image plugin to resize and crop images.
+However, if you don't have control over the images that to be displayed in your App (for example, because they originate from a remote API), you can use the *Image* plugin to resize and crop the images.
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/image.md:7]: Both agreed.

File: docs/plugins/image.md [hunk 7]
```diff
@@ -1,79 +1,83 @@
...
 
 ## Usage
 
-Whenever you need to resize image, import the Image plugin from the Lightning SDK
+Import the Image plugin from the Lightning SDK in components where you want to resize an image:
 
-```js
+```
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/image.md:13]: Added 'js' suffix at all occurrences.

File: docs/plugins/image.md [hunk 8]
```diff
@@ -1,79 +1,83 @@
...
 
-## Image quality
+## Image Quality
+
+To increase the performance on lower-end boxes –especially those with limited GPU memory– there is a platform setting `image.quality` which enables you to control the *image quality*.
 
-To increase performance on lower end boxes, especially those with limited GPU memory, there exists a `platform` setting to control the _image quality_.
-Depending on this setting the images returned by the image server will be _smaller_ than actually displayed on the screen.
-Lightning will then _stretch_ the images to fit the desired dimensions. This will save on used GPU memory, but off course impacts the visual quality of the images.
+Depending on this setting, the images that are returned by the image server may be *smaller* than actually displayed on the screen.
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/image.md:72]: Agreed.

File: docs/plugins/image.md [hunk 9]
```diff
@@ -1,79 +1,83 @@
...
+Lightning *stretches* the images to fit them within the desired dimensions.
 
-Please note that within the context of the Metrological AppStore this setting is defined as a platform setting _outside_ of the control of the App.
-You can however experiment with this during local development via `settings.json`:
+> Although this setting saves on used GPU memory, it also impacts the visual quality of the images.
 
-```json
+Within the context of the Metrological Application Platform, this setting is defined as a platform setting *outside* the control area of your App.
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/image.md:77]: Modified text accordingly.

File: docs/plugins/image.md [hunk 10]
```diff
@@ -1,79 +1,83 @@
...
+However, you can experiment with this setting during local development via **settings.json** (located in the root of your project):
+
+```
```
[the remark this answers is not in the document]
Comment (LBakkerTDS) [reply docs/plugins/image.md:80]: Added 'json' suffix.

File: docs/plugins/language.md (hunk 1 above)
Comment (LBakkerTDS) [reply docs/plugins/language.md:22]: Corrected at all occurrences.

File: docs/plugins/language.md (hunk 2 above)
Comment (LBakkerTDS) [reply docs/plugins/language.md:38]: Corrected accordingly.

**Pull request closed without merging**